test_train_local.py

```
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+"""
+ローカルでトレーニングスクリプトの初期化部分をテストするスクリプト
+"""
+
+import sys
+import os
+
+# プロジェクトルートをPythonパスに追加
+sys.path.insert(0, os.path.dirname(os.path.abspath(__file__)))
+
+from src.train import train_model
```

Ruff (0.11.9)

`src.train.train_model` imported but unused

Remove unused import: `src.train.train_model`

(F401)